src/static/styles/partials/_settings.css

```
@@ -236,10 +317,46 @@
+.settings-panel .buttons-container {
+  position: sticky;
+  bottom: 16px;
+  z-index: 6;
+  align-items: center;
+  justify-content: flex-end;
+  gap: 12px;
+  padding-top: 12px;
+  background: linear-gradient(180deg, transparent, color-mix(in srgb, var(--bg) 94%, transparent) 22%);
+}
+
+.settings-panel .buttons-container #settings-form-status {
```

Stylelint (17.7.0)

[error] Duplicate selector ".settings-panel .buttons-container `#settings-form-status`", first used at line 330 (no-duplicate-selectors)

(no-duplicate-selectors)